ha/k8s_setup/ha_start.py

```diff
@@ -53,8 +53,19 @@ def handle_signal(signum, frame):
     signal.signal(signal.SIGSEGV, handle_signal)
     signal.signal(signal.SIGTERM, handle_signal)
 
-    driver_process = Popen(['/usr/bin/python3', service_entry_mapping[args.services]], shell=False, stdout=PIPE, stderr=PIPE) # nosec
+    driver_process = Popen(['/usr/bin/python3', service_entry_mapping[args.services], "--start"], shell=False, stdout=PIPE, stderr=STDOUT) # nosec
+
     sys.stdout.write(f"The driver process with pid {driver_process.pid}, and args {driver_process.args} started successfully.")
+
+    # Poll process.stdout to show stdout live
```

Comment:
changes looks good but please check once if SIGTERM scenario is working as expected

Reply:
yes it is working